audius-cli

```diff
@@ -903,6 +906,46 @@ def launch_notifications(ctx):
     )
 
 
+@cli.command()
+@click.pass_context
+def launch_pedalboard(ctx):
+    lock(ctx, "docker")
+
+    pedals = ["trending-challenge-rewards"]
```

Comment:
we could just leave this out and "up" the whole docker compose file?

Reply:
should the pedals be passed in as args? fine either way for now since there's just one

Reply:
Was thinking that too, like maybe a list that the provider chooses to stand up?

Reply:
Yeah I'm thinking the click multiple args is the way to go, then maybe provide a base and/or all option? Some plugins are required like es-indexer

Reply:
I'm a little concerned about the naming of pedals vs. plugins. Can we call this plugins here, but keep the "framework" as pedalboard?  (like we have `register notifications` right now)

Also, agree w/ Isaac on the args! I think we can adopt the same thing we did for `register` https://github.com/AudiusProject/audius-docker-compose/blob/dev/audius-cli#L909